lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,28 @@
-import React, { Component } from 'react';
-import Card from './Card';
+import React, { Component } from "react";
+import Card from "./Card";
+import PropTypes from "prop-types";
+import * as cardstyle from "../../styles/cardsstyle";
 
 const Cards = props => {
   return (
-    <div className="cards-container">
-      {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
-    </div>
-  )
-}
+    <cardstyle.CardsContainer>
+      {props.cards.map((card, index) => (
+        <Card card={card} key={index} />
+      ))}
```

Comment:
Good job making list of `Cards` components and giving a unique `key` to each child